com.unity.cinemachine/Editor/Editors/CinemachineSplineDollyEditor.cs

```diff
@@ -39,6 +42,18 @@ void TrackSpline(SerializedProperty p)
                     noSpline = targets[i] != null && ((CinemachineSplineDolly)targets[i]).Spline == null;
                 noSplineHelp.SetVisible(noSpline);
             }
+            
+            // Store the initial value
+            PathIndexUnit previousUnits = Target.positionUnitsBackingField;
+            ux.TrackPropertyValue(positionUnitsProp, TrackPositionUnits);
+            void TrackPositionUnits(SerializedProperty p)
+            {
+                PathIndexUnit newUnits = (PathIndexUnit) p.enumValueIndex;
+                
+                Target.UpdateDistanceForPositionUnits(previousUnits, newUnits);
```

Comment:
This code is problematic because it doesn't handle undo or multi-object selection

Reply:
Good point, bad oversight of mine!